src/app/api/v1/applications/[appSlug]/users/verify/route.ts

```diff
@@ -0,0 +1,51 @@
+import { NextResponse } from "next/server";
+import { StatusCodes } from "http-status-codes";
+import { verifyEmailOtp } from "@/lib/emails/auth/verifyEmailOtp";
+import { userModel } from "@/models";
+import { refreshAccountSession } from "@/lib/auth/accounts/refreshAccountSession";
+import { getAppIdBySlug } from "@/lib/app";
+import { createUserAccount } from "@/lib/auth/accounts/createUserAccount";
+
+export async function POST(req: Request, { params: { appSlug } }) {
```

Comment:
add validation pls. Use `src/app/api/v1/applications/[appSlug]/tickets/[id]/supply/route.ts` as the ref